packages/addons/addon-depends/python/libsodium/package.mk

```diff
@@ -0,0 +1,32 @@
+###############################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2017 Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="libsodium"
+PKG_VERSION="1.0.13"
+PKG_SHA256="83c3017600bf18b349bfcf97cb8bf07e97ee2793743e860a83a0f08a2c251593"
+PKG_ARCH="any"
+PKG_LICENSE="ISC"
+PKG_SITE="https://libsodium.org/"
+PKG_URL="https://github.com/jedisct1/libsodium/archive/$PKG_VERSION.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_SECTION="security"
+PKG_SHORTDESC="Sodium is a modern, easy-to-use software library for encryption, decryption, signatures, password hashing and more."
+PKG_LONGDESC="Sodium is a modern, easy-to-use software library for encryption, decryption, signatures, password hashing and more."
+
+PKG_IS_ADDON="no"
```

Comment:
Please remove above 2 lines and update Copyright ("2017-present" etc.) in licence (#2117). Thanks!